plugins/modules/cloud/misc/proxmox_tasks_info.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
Please create a new PR for this module. Originally you had proxmox_interface and proxmox_interfaces in this PR, which is kind of OK, but in general we do not accept multiple modules per PR.

(Also this module has no tests.)

Reply:
Created  new PR  #3226 for the `proxmox_tasks_info` module. I have some difficulties with mocking the ProxmoxAPI though.